nix/mobile/android/build.nix

```
@@ -154,8 +157,9 @@ in stdenv.mkDerivation rec {
       --no-watch-fs \
       --no-build-cache \
       -Dmaven.repo.local='${deps.gradle}' \
-      -PversionCode=${toString buildNumber} \
+      -PversionCode=${toString versionCode} \
       -PcommitHash=${commitHash} \
+      -PhermesEnabled=${hermesEnabled} \
```

I think we should get rid of this:
```suggestion
      -PversionCode=${toString versionCode} \
      -PcommitHash=${commitHash} \
      -PhermesEnabled=${hermesEnabled} \
```
And just set env vars directly:
```nix
  ORG_GRADLE_PROJECT_versionCode = versionCode;
  ORG_GRADLE_PROJECT_commitHash = commitHash;
  ORG_GRADLE_PROJECT_publicUrl = publicUrl;
  ORG_GRADLE_PROJECT_hermesEnabled = hermesEnabled;
```
Probably in place of the lines 87-89, which you missed.